site/src/pages/_document.page.tsx

```diff
@@ -54,10 +54,6 @@ class MyDocument extends Document {
             href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800;900&display=swap"
             rel="stylesheet"
           />
-          <script
-            src="https://kit.fontawesome.com/31605611a9.js"
-            crossOrigin="anonymous"
-          />
```

Comment:
What's the thinking behind moving this script into the body? To increase page-load speed?

The font [awesome docs](https://fontawesome.com/v5.15/how-to-use/javascript-api/setup/configuration) seem to indicate to place this into the header instead, so I'd be inclined to keep it here for now.

Either-way this isn't a big deal as I think we're going to change how these icons are loaded soon anyway for two reasons:
1. so the SVGs aren't fetched by the client, but statically added when the page is built by NextJs
2. so that the SVGs are treated as react children (currently react is not aware of them, so we can't do things like update the styling of an SVG element using JS)